Apps/WebClient/src/ClientApp/src/components/services/OrganDonorDetailsCard.vue

```diff
@@ -18,7 +18,7 @@ import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
 import { ILogger } from "@/services/interfaces";
 import SnowPlow from "@/utility/snowPlow";
 
-library.add(faCircleInfo, faDownload);
+library.add(faDownload, faInfoCircle);
```

Comment:
`faCircleInfo` and `faInfoCircle` are not equivalent. Not sure if this was an intentional change, but its not necessary to nit pick this. So I'm not rejecting this.